tests/vmi_configuration_test.go

```diff
@@ -446,7 +447,10 @@ var _ = Describe("[sig-compute]Configurations", func() {
 				Expect(domXml).To(ContainSubstring("driver name='vhost' queues='3'"))
 				// make sure that there are not block queues configured
 				Expect(domXml).ToNot(ContainSubstring("cache='none' queues='3'"))
-			})
+			},
+				Entry("[test_id:1665]VM with masquerade binding", func(vmi *v1.VirtualMachineInstance) {}),
+				Entry("VM with autoattached interfaces", resetInterfaces),
```

Comment:
Please just pass another VM. This `func` is confusing.

Reply:
Tried to do it more explicit,
we can't send the `vmi` directly as we spoke offline, because it is created only when `BeforeEach` is called,
and not when the spec tree is built.

The alternative is to split to couple contexts so we won't create the vmi in the BeforeEach that isn't used.

hope you will like the change